script/bin/genesis.rs

```diff
@@ -123,8 +123,7 @@ pub async fn main() -> Result<()> {
     genesis_config.source_chain_id = source_chain_id;
     genesis_config.sync_committee_hash = format!("0x{:x}", sync_committee_hash);
     genesis_config.header = format!("0x{:x}", finalized_header);
-    genesis_config.execution_state_root =
-        format!("0x{:x}", execution_state_root_proof.execution_state_root);
+    genesis_config.execution_state_root = format!("0x{:x}", helios_client.store.finalized_header.beacon().state_root);
```

Comment:
This doesn't seem right. This is the `beacon_state_root`. This should be the execution payload's state root.